.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
+
+before_install:
+  - sudo add-apt-repository --yes ppa:kalakris/cmake
```

Comment:
http://docs.travis-ci.com/user/languages/c/ lists cmake as one of the built-in tools